thirdparty/aqt_settings.ini

```diff
@@ -0,0 +1,249 @@
+# Copied and modified from:
+# https://github.com/miurahr/aqtinstall/blob/v3.1.16/aqt/settings.ini
+# Main changes:
+# 1. Change hash_algorithm from sha256 to sha1
+# 2. Move https://master.qt.io/ from fallbacks to trusted_mirrors
+
+[DEFAULTS]
+
+[aqt]
+concurrency: 4
+baseurl: https://download.qt.io
+7zcmd: 7z
+print_stacktrace_on_error: False
+always_keep_archives: False
+archive_download_location: .
+min_module_size: 41
+
+[requests]
+connection_timeout: 3.5
+response_timeout: 30
+max_retries_on_connection_error: 5
+retry_backoff: 0.1
+max_retries_on_checksum_error: 5
+max_retries_to_retrieve_hash: 5
+hash_algorithm: sha1
```

Comment:
here's the original
https://github.com/miurahr/aqtinstall/blob/8c4bf2ed70709b243292118cd60c19eff5480013/aqt/settings.ini#L19